src/desktop/apps/article/components/InfiniteScrollArticle.tsx

```diff
@@ -97,7 +112,7 @@ export default class InfiniteScrollArticle extends React.Component {
         return (
           <Waypoint
             onEnter={this.fetchNextArticles}
-            threshold={2.0}
+            // threshold={2.0}
```

Comment:
Any particular reason why this is commented out?

Reply:
I commented this out while working on it -- after converting to typescript it looks like the Waypoint library no longer accepts this property.  Tested it and the UI seems to act as expected without it.